fabtests/multinode/src/core.c

```diff
@@ -259,15 +307,15 @@ static int multinode_run_test()
 		while (!state.all_completions_done ||
 				!state.all_recvs_posted ||
 				!state.all_sends_posted) {
-			ret = multinode_post_rx();
+			ret = method.send();
 			if (ret)
 				return ret;
 
-			ret = multinode_post_tx();
+			ret = method.send();
 			if (ret)
 				return ret;
 
-			ret = multinode_wait_for_comp();
+			ret = method.send();
```

Comment:
This calls send 3 times, which a later patch fixes.  Apply the correct code to this patch.